json/tasmocompiler.json

```
@@ -0,0 +1,34 @@
+{
+    "name": "TasmoCompiler",
+    "slug": "tasmocompiler",
+    "categories": [
+        0
+    ],
+    "date_created": "2025-02-10",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": 3000,
+    "documentation": "https://github.com/benzino77/tasmocompiler/blob/master/README.md",
+    "website": "https://github.com/benzino77/tasmocompiler",
+    "logo": "https://github.com/benzino77/tasmocompiler/raw/master/docs/logo/tasmocompiler-full.svg",
```

Is the symbol not better for the website?
```suggestion
    "logo": "https://github.com/benzino77/tasmocompiler/raw/master/docs/logo/tasmocompiler-symbol.svg",
```